code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -221,6 +236,14 @@
 		/obj/item/stack/cable_coil = 2,
 		/obj/item/stock_parts/subspace/filter = 1)
 
+/obj/item/circuitboard/machine/pacman/crude
+	name = "CAVEMAN-type Generator (Machine Board)"
+	icon_state = "engineering"
+	build_path = /obj/machinery/power/port_gen/pacman/crude
+	req_components = list(/obj/item/stack/ore/iron = 2,
```

Comment:
You'll need a capacitory in here or else you'll get zero power output in port_gen.dm when it grabs the temp rating.